src/main/java/task/TaskList.java

```diff
@@ -0,0 +1,120 @@
+package task;
+
+import util.Ui;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    public List<Task> list;
```

Comment:
Class variables should never be declared as public.
![image](https://user-images.githubusercontent.com/80502407/151960241-82ced349-a4d9-4306-8d9c-7a7229d4306d.png)